src/test/java/com/solvd/carina/demo/TestWithThreads.java

```diff
@@ -0,0 +1,79 @@
+package com.solvd.carina.demo;
+
+import java.util.concurrent.*;
+
+public class TestWithThreads {
```

Comment:
Have you tried configuring Maven/Testng or Carina configs to run methods  in parallel?